src/components/TransactionSender/index.ts

```diff
@@ -18,12 +19,17 @@ import {
 } from 'types/enums';
 import { setNonce } from 'utils';
 
-const TransactionSender = () => {
+export interface TransactionSenderType {
+  sendSignedTransactionsAsync?: SendSignedTransactionsAsyncType;
+}
+
+export const TransactionSender = ({
+  sendSignedTransactionsAsync: customSendTxAsync
+}: TransactionSenderType) => {
   const account = useSelector(accountSelector);
   const signedTransactions = useSelector(signedTransactionsSelector);
-  const { sendSignedTransactionsAsync } = useContext(
-    OverrideDefaultBehaviourContext
-  );
+
+  const sendSignedTransactionsAsync = customSendTxAsync ?? defaultSendSignedTxs;
```

Comment:
?? will fail, due to microbundle, use || instead

Reply:
dapp-core is now on `tsdx`, and working ✅